.github/workflows/ecp-data-vis-sdk.yml

```diff
@@ -20,6 +20,8 @@ jobs:
       uses: hendrikmuhs/ccache-action@v1.2
       with:
         key: ubuntu-gcc-ccache
+    - name: Submodules
```

Comment:
Try moving the submodule update step to the actions/checkout@3 section...I am not sure if that would work, but the error it throughs makes no sense to me since we already run the ownership cleanup.

```yaml
steps:
    - uses: actions/checkout@v3
      with:
        submodules: recursive
```

Reply:
the other problem may be with using `@v3` for the checkout tool. I didn't think anything of the update since it still worked, but that is a difference between the two pipelines updated here.